### ISSUE TITLE
Devtools are included in Vite bundle

### ISSUE THREAD

**Issue by Xenfo** [jotaijs/jotai-devtools#54]
The devtools aren't excluded and end up adding 1mb to my bundle size.

**Comment by arjunvegda** [comment]
What bundler are you using?

**Comment by Xenfo** [comment]
I think Vite uses Rollup under the hood

**Comment by arjunvegda** [comment]
We're investigating similar issues for [CRA](https://github.com/jotaijs/jotai-devtools/pull/44#issuecomment-1474866468) and [NextJS](https://github.com/jotaijs/jotai-devtools/discussions/53#discussioncomment-5511954). Could you try wrapping the `<DevTools/>` component like below and see if that does the trick?

```ts
const DevOnlyJotaiDevTools = () => process.env.NODE_ENV !== 'production' ? <DevTools/> : null;
```

**Comment by Xenfo** [comment]
That does fix it

**Comment by arjunvegda** [comment]
Hey @Xenfo 

Could you help us test this build? We're expecting tree-shaking to automatically work without needing to wrap the component with `process.env` on userland.

```shell
yarn add jotai-devtools@https://pkg.csb.dev/jotaijs/jotai-devtools/commit/2b2603e6/jotai-devtools/_pkg.tgz
```

**Comment by Xenfo** [comment]
Sure, I'll test it out in a few days. Thanks.

**Comment by arjunvegda** [comment]
That's great news! We'll release the fix as part of `0.5.0`
